src/models/array.rs

```diff
@@ -116,96 +96,106 @@ impl ArrayModel {
-    /// 
+    ///
     /// let input = [
     ///  [0,1,1,0],
     ///  [1,2,2,1],
     ///  [1,2,2,1],
     ///  [0,1,1,0]
     /// ];
-    /// 
+    ///
     /// let game = Matrix::from(input);
-    /// 
+    ///
     /// assert_eq!(game.as_matrix(), input);
     /// ```
-    /// 
+    ///
     fn from(board: MatrixBoard) -> Self {
         // TODO: Implement macro
         ArrayModel {
             board: [
-                board[0][0], board[0][1], board[0][2], board[0][3],
-                board[1][0], board[1][1], board[1][2], board[1][3],
-                board[2][0], board[2][1], board[2][2], board[2][3],
-                board[3][0], board[3][1], board[3][2], board[3][3]
-            ]
+                board[0][0],
+                board[0][1],
+                board[0][2],
+                board[0][3],
+                board[1][0],
+                board[1][1],
+                board[1][2],
+                board[1][3],
+                board[2][0],
+                board[2][1],
+                board[2][2],
+                board[2][3],
+                board[3][0],
+                board[3][1],
+                board[3][2],
+                board[3][3],
+            ],
         }
     }
 }
 
 impl From<ArrayBoard> for ArrayModel {
     /// Sets the board state based on the given array
-    /// 
+    ///
     /// # Examples
     /// ```
     /// use game_2048_model::models::{Model, ArrayModel};
-    /// 
+    ///
     /// let input = [
     ///     0,1,1,0,
     ///     1,2,2,1,
     ///     1,2,2,1,
     ///     0,1,1,0
     /// ];
-    /// 
+    ///
     /// let game = ArrayModel::from(input);
-    /// 
+    ///
     /// assert_eq!(game.as_array(), input);
     /// ```
-    /// 
+    ///
     fn from(board: ArrayBoard) -> Self {
-        ArrayModel {
-            board: board
-        }
+        ArrayModel { board: board }
     }
 }
 
 impl Model for ArrayModel {
     /// Create a new instance of the game board filled with zeros
-    /// 
+    ///
     /// # Examples
-    /// 
+    ///
     /// ```
     /// use game_2048_model::models::{Model, ArrayModel};
-    /// 
+    ///
     /// let game = ArrayModel::new();
     /// ```
-    /// 
+    ///
     fn new() -> ArrayModel {
         ArrayModel {
             board: [0; BOARD_SIZE * BOARD_SIZE],
         }
     }
 
     /// Slide and merge the numbers towards a direction
-    /// 
+    ///
     /// # Examples
-    /// 
+    ///
     /// ```
     /// use game_2048_model::models::{ArrayModel, Directions, Model};
     /// use rand::thread_rng;
-    /// 
+    ///
     /// let mut game = ArrayModel::from([
-    ///     2,1,16,2,
-    ///     4,1,8,2,
-    ///     0,0,8,2,
-    ///     4,0,4,2
+    ///     2,1,5,2,
+    ///     3,1,4,2,
+    ///     0,0,4,2,
+    ///     3,0,3,2
     /// ]);
     /// game.slide(Directions::Down);
-    /// 
+    ///
     /// assert_eq!(game.as_array(), [
     ///     0,0,0,0,
-    ///     0,0,16,0,
-    ///     2,0,16,4,
-    ///     8,2,4,4
```

Comment:
Suggest we keep the pattern for from the old example and lower convert all values instead, i.e. 2 => 1, 4 => 2, 8 => 3 and 16 => 4.